src/node_modules/_file-explorer.js

```diff
@@ -0,0 +1,292 @@
+var setval = require('setval')
```

Comment:
i can improve the documentation.
it basically turns:

```js
var data = { }
;['a/b/c', 'a/b/d', 'x/y/z'].forEach(function (path) {
  setval(data, path, '')
})
console.log(data) // into
/*
 { a: { b: { c: '', d: '' }, x: { y: { z: '' } } }
*/
````
setval by default takes '/' as a seperator but it can also take a custom one (like ".") and splits the path and sets them on a nested object.

The readme of the npm module tries to show this via lots of examples